modular_skyrat/modules/customization/modules/surgery/organs/silicon_alt_brains.dm

```diff
@@ -0,0 +1,70 @@
+/obj/item/mmi/posibrain/circuit
+	name = "compact AI circuit"
+	desc = "A compact circuit, perfectly dimensioned to fit in the same slot as a cyborgs positronic brain."
+	icon = 'modular_skyrat/master_files/icons/obj/alt_silicon_brains.dmi'
+	icon_state = "circuit"
+	base_icon_state = "circuit"
+	inhand_icon_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+
+	// It pains me to copy-paste so much, but I can't do it any other way
+	begin_activation_message = "<span class='notice'>You carefully locate the manual activation switch and start the compact AI circuit's boot process.</span>"
+	success_message = "<span class='notice'>The compact AI circuit pings, and its lights start flashing. Success!</span>"
+	fail_message = "<span class='notice'>The compact AI circuit buzzes quietly, and the golden lights fade away. Perhaps you could try again?</span>"
+	new_mob_message = "<span class='notice'>The compact AI circuit chimes quietly.</span>"
+	recharge_message = "<span class='warning'>The compact AI circuit isn't ready to activate again yet! Give it some time to recharge.</span>"
+
+/obj/item/organ/internal/brain/ipc_positron/circuit
+	name = "compact AI circuit"
+	desc = "A compact and extremely complex circuit, perfectly dimensioned to fit in the same slot as a synthetic-compatible positronic brain. It is usually slotted into the chest of synthetic crewmembers."
+	icon = 'modular_skyrat/master_files/icons/obj/alt_silicon_brains.dmi'
+	icon_state = "circuit-occupied"
+	inhand_icon_state = "electronic"
+	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
+
+/obj/item/organ/internal/brain/ipc_positron/mmi
+	name = "compact man-machine interface"
+	desc = "A compact man-machine interface, perfectly dimensioned to fit in the same slot as a synthetic-compatible positronic brain. Unfortunately, the brain seems to be permanently attached to the circuitry, and it seems relatively sensitive to it's environment. It is usually slotted into the chest of synthetic crewmembers."
+	icon = 'modular_skyrat/master_files/icons/obj/surgery.dmi'
+	icon_state = "mmi-ipc"
+
+// CODE THAT ACTUALLY APPLIES THE BRAINS.
+// See modular_skyrat/master_files/code/modules/client/preferences/brain.dm for Synth/IPC application.
+
+/proc/prefs_get_brain_to_use(value, is_cyborg = FALSE)
+	switch(value)
+		if(ORGAN_PREF_POSI_BRAIN)
+			return is_cyborg ? /obj/item/mmi/posibrain : null
```

Comment:
just to make sure, would `null` here result in no brain or brain/ipc_positron?

Reply:
Second, though I'm going to clean up the logic, cause this is a bad, snowflakey way of doing it.